### PR TITLE
Added styling

### PR DESCRIPTION
**Pull request by Jessica-Mach**
We've got style!

### CODE REVIEW

File: client/components/Navbar.js [hunk 1]
```diff
@@ -30,23 +30,45 @@ const NavBar = props => {
...
+      </li>
+    ) 
   })
-
+  
   return(
     <div>
+      <div id="top-header">
+        <div>
+          {/* <img src={require("../assets/images/logo.png")} alt="logo" /> */}
```
Comment (dovakeen118) [inline client/components/Navbar.js:44]: remove if not used!

File: client/components/PetTile.js [hunk 2]
```diff
@@ -5,13 +5,15 @@ import { Link } from "react-router-dom";
...
   const { id, name, imageUrl, age, vaccinationStatus } = props.pet;
   return (
-    <div >
-      <Link to={`/pets/${props.type}/${id}`}>{name}</Link>
+    <div className="tile">
       <Link to={`/pets/${props.type}/${id}`} >
         <img src={imageUrl} alt={props.type} />
       </Link>
+      <h4>
+        <Link to={`/pets/${props.type}/${id}`}>{name}</Link>
```
Comment (dovakeen118) [inline client/components/PetTile.js:13]: these 2 Links are to the same route, we can condense to one Link wrapping around both pieces of info!

File: client/components/PetTypeTile.js [hunk 3]
```diff
@@ -10,13 +10,13 @@ const PetTypeTile = (props) => {
...
-        <div>
-            <h3>
-                <Link to={`/pets/${type}`}>{type}</Link>
-            </h3>
+        <div className="tile">
             <Link to={`/pets/${type}`}>
                 <img src={imageUrl} alt={type}></img>
             </Link>
+            <h4>
+                <Link to={`/pets/${type}`}>{type}</Link>
```
Comment (dovakeen118) [inline client/components/PetTypeTile.js:18]: Here too, the Links are the same so we don't need to repeat

**Review by dovakeen118 — COMMENTED** [review]
Couple teeny tiny things!

File: client/components/Navbar.js (hunk 1 above)
Comment (Jessica-Mach) [reply client/components/Navbar.js:44]: Serge may add this later, but he'll make a new branch if he does.

File: client/components/PetTile.js (hunk 2 above)
Comment (Jessica-Mach) [reply client/components/PetTile.js:13]: Wow! I didn't know you could do that!

**Pull request merged**